### ISSUE TITLE
Dropdown bug

### ISSUE THREAD

**Issue by BrettCleary** [HyperPlay-Gaming/hyperplay-desktop-client#441]
There is a bug in the dropdown to connect my wallet, it shows options that shouldn't show or should be enabled when I'm not connected to my wallet. We should improve the flow of these log-in options.

![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/27568879/3fe5d25b-4cd7-4fe1-bd62-8959fe09191e)

**Comment by BrettCleary** [comment]
We need steps to reproduce this one

**Comment by nyghtstalker** [comment]
Steps to Reproduce:


```
(13:50:02) INFO:    [Legendary]:        Refreshing library...
(13:50:02) INFO:    [Backend]:          Loading MetaMask extension
(13:50:02) INFO:    [Connection]:       Connectivity: check-online
(13:50:02) INFO:    [Connection]:       Pinging external endpoints
(13:50:02) INFO:    [Backend]:          Creating abort controller for legendary-version
(13:50:02) INFO:    [Legendary]:        Running command: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\legendary --version
(13:50:04) INFO:    [Legendary]:        Legendary location: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\legendary
(13:50:04) INFO:    [Gog]:              GOGDL location: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\gogdl
(13:50:04) INFO:    [Gog]:              GOGDL location: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\gogdl
(13:50:04) INFO:    [Backend]:          MetaMask extension loaded
(13:50:04) WARNING: [Backend]:          Protocol already registered.
(13:50:04) INFO:    [Backend]:          App is starting in headless mode = undefined isCLINoGui = false startInTray = undefined
(13:50:04) INFO:    [Connection]:       Connectivity: online
(13:50:04) INFO:    [Backend]:          User Not Found, removing it from Store
(13:50:05) INFO:    [Backend]:          Auto Updater found version: 0.7.1 released on Wed Oct 04 2023 17:17:14 GMT-0700 (Pacific Daylight Time) with name 0.7.1
(13:50:06) INFO:    [WineDownloader]:   Updating wine versions info
(13:50:06) INFO:    [WineDownloader]:   Fetching upstream information...
(13:50:06) INFO:    [Backend]:          Deleting abort controller for legendary-version
(13:50:06) INFO:    [Backend]:          Creating abort controller for gogdl-version
(13:50:06) INFO:    [Gog]:              Running command: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\gogdl --version
(13:50:08) INFO:    [Backend]:          Frontend Ready
(13:50:08) INFO:    [Backend]:          Loading Screen Ready
(13:50:08) INFO:    [Frontend]:         Refreshing Library
(13:50:09) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(13:50:09) INFO:    [Gog]:              Found 0 game(s) to update
(13:50:10) INFO:    [WineDownloader]:   wine versions updated
(13:50:11) ERROR:   [Backend]:          Could not find default wine version
(13:50:11) INFO:    [Backend]:          Deleting abort controller for gogdl-version
(13:50:13) INFO:    [Backend]:          Starting the Download Queue
(13:50:25) INFO:    [Backend]:          

HyperPlay Version: 0.7.1
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.6
OS: Microsoft Windows 10 Pro KERNEL: 10.0.19045 ARCH: x64
CPU: Intel Core™ i7-6600U @2.6 
RAM: Total: 7.84 GiB Available: 993.36 MiB
GRAPHICS: GPU0: Intel(R) HD Graphics 520 VRAM: 1024MB  


(14:37:30) INFO:    [Connection]:       Connectivity: offline
(14:37:34) INFO:    [Connection]:       Connectivity: check-online
(14:37:34) INFO:    [Connection]:       Pinging external endpoints
(14:37:34) INFO:    [Connection]:       Connectivity: online
(14:39:14) INFO:    [Backend]:          Creating abort controller for legendary-version
(14:39:14) INFO:    [Legendary]:        Running command: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\legendary --version
(14:39:17) INFO:    [Backend]:          Creating abort controller for gogdl-version
(14:39:17) INFO:    [Gog]:              Running command: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\gogdl --version
(14:39:19) INFO:    [Backend]:          Deleting abort controller for legendary-version
(14:39:19) INFO:    [Backend]:          Creating abort controller for 98bc04bc842e4906993fd6d6644ffb8d
(14:39:19) INFO:    [Legendary]:        Checking if EOS Overlay is enabled: C:\Users\Amy\AppData\Local\Programs\hyperplay\resources\app.asar.unpacked\build\bin\win32\legendary eos-overlay info
```
